src/z3c/checkversions/base.py

```diff
@@ -16,14 +16,10 @@
 from setuptools import package_index
 
 
-_final_parts = '*final-', '*final'
 def _final_version(parsed_version):
     """Function copied from zc.buildout.easy_install._final_version
```

Comment:
Is the docstring still accurate now that you've replaced the implementation?

Reply:
Yes it is, I copied the implementation from the current version of `zc.buildout`.